src/OpenTelemetry.Exporter.NewRelic.Samples/SampleAspNetFrameworkApp/Controllers/WeatherForecastController.cs

```diff
@@ -16,7 +16,7 @@ public class WeatherForecastController : ApiController
         };
 
         [HttpGet]
-        public async Task<IEnumerable<WeatherForecast>> Get()
+        public IEnumerable<WeatherForecast> Get()
         {
             var span = WebApiApplication.OTTracer.StartRootSpan("/WeatherForecastController/Get");
 
```

Comment:
is this correct?   I would have thought that the opentelemetry provider for ASP.NET would create the trace/span for us already.   OpenTelemetry has the concept of currentSpan.......